src/NzbDrone.Api/Series/SeasonResource.cs

```diff
@@ -41,7 +41,7 @@ public static List<SeasonResource> ToResource(this IEnumerable<Season> models)
 
         public static List<Season> ToModel(this IEnumerable<SeasonResource> resources)
         {
-            return resources.Select(ToModel).ToList();
+            return resources?.Select(ToModel).ToList();
```

Comment:
I'm not opposed to the ?. operator in general, but in this case having a separate `if` makes it more clear that it can accept and emit a null.